scripts/crd_template.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: exampleresources.CRD_COUNT.example.com
+spec:
+  group: CRD_COUNT.example.com
+  names:
+    kind: ExampleResource
+    listKind: ExampleResourceList
+    plural: exampleresources
+    singular: exampleresource
+  scope: Namespaced
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
```

Comment:
I propose we use the HR spec and all its versions in here, to have a closer load to what you’re actually running in your production clusters. This CRD, as it now, it’s 1kb memory at max.